server/src/main/java/com/linecorp/centraldogma/server/internal/mirror/GitMirror.java

```diff
@@ -395,6 +398,15 @@ private GitWithAuth openGit(File workDir) throws Exception {
         }
     }
 
+    private static String generateCommitDetail(String abbrId, RevCommit headCommit) {
+        final String newLine = System.lineSeparator();
+        return "The commit information of " + abbrId + ": " + newLine +
```

Comment:
Perhaps we don't need to add `abbrId` because we already have it in the summary? cc @ikhoon

Reply:
Thanks, @minwoox, I removed!